priam/src/main/java/com/netflix/priam/backup/BackupVerification.java

```diff
@@ -114,11 +115,11 @@ public BackupVerificationResult verifyBackup(List<BackupMetadata> metadata, Date
         //Download meta.json from backup location and uncompress it.
         List<String> metaFileList = new ArrayList<>();
         try {
-            bkpStatusFs.download(metas.get(0), new FileOutputStream("tmp_meta.json"));
+            bkpStatusFs.download(metas.get(0), new FileOutputStream(config.getDataFileLocation() + File.separator + "tmp_meta.json"));
```

Comment:
Can you use https://docs.oracle.com/javase/7/docs/api/java/nio/file/Path.html to make this cleaner?

e.g. https://docs.oracle.com/javase/7/docs/api/java/nio/file/Paths.html#get(java.net.URI)